lib/passport-mock/factories/legalshield.js

```diff
@@ -0,0 +1,88 @@
+var utils = require('./utils');
+var faker = require('faker');
+
+module.exports = function Legalshield (params) {
+  params = params || { };
+  params.address = params.address || {};
+  params.fulfillment_partner = params.fulfillment_partner || {};
+  params.emergency_contact = params.emergency_contact || {};
+  params.access_control = params.access_control || {};
+
+  var id = (params.id || utils.makeNumber(10)) ;
+  var firstName = (params.firstName || faker.name.firstName());
+  var lastName = (params.lastName || faker.name.lastName());
+  var displayName = (params.displayName || firstName + ' ' + lastName);
+  var email = (params.email || utils.makeEmail(firstName, lastName));
+  var json = {
+    id: id,
+    account_type: "Member",
+    provider: "legalshield",
+    displayName: displayName,
+    name: {
+      givenName: firstName,
+      middleName: "",
+      familyName: lastName
+    },
+    emails: [
+      {
+      value: email,
+      type: "home"
+      }
+    ],
+    _json: {
+      type: "LegalShield",
+      first_name: firstName,
+      last_name: lastName,
+      membership_number: id,
+      day_time_phone_number: null,
+      home_phone_number: null,
+      cell_phone_number: (params.cellphone || faker.phone.phoneNumber()),
+      last_4_of_tax_id: (params.ssn || utils.makeNumber(4)),
+      date_of_birth: (params.dob || faker.date.past()),
+      fulfillment_partner: {
+        phone: (params.fulfillment_partner.phone || faker.phone.phoneNumber()),
```

Comment:
won't this blow up if the params doesn't have a `fulfillment_partner` property? you'd call a property on undefined.